user-guide/Troubleshooting/Known_issues/KI_Cube_connection_issue_alarm_tabs.md

```diff
@@ -0,0 +1,35 @@
+---
+uid: KI_Cube_connection_issue_alarm_tabs
+---
+
+# Cube freezes on 'Connected!' loading screen when no alarm tabs are displayed
+
+## Affected versions
+
+DataMiner versions 10.3.9 and 10.3.10.
```

Comment:
```suggestion
- DataMiner versions 10.3.9 and 10.3.10.

- Any versions from DataMiner 10.2.0 onwards using automatic client updates
```

Is this better, @AlexanderGorbunovSkyline?

Reply:
Looks good to me, thanks!